yarn.lock

```diff
@@ -38,13 +38,14 @@
     tslib "^2.3.0"
     zen-observable-ts "^1.2.5"
 
-"@aptos-labs/aptos-names-connector@^0.0.1":
-  version "0.0.1"
-  resolved "https://registry.yarnpkg.com/@aptos-labs/aptos-names-connector/-/aptos-names-connector-0.0.1.tgz#51f5ea6f6fc80b679ed4b03a29f35623231bcd25"
-  integrity sha512-HTVqbnsaKyfaqGdf7fC9dJsj8hr9aLS4mB5XUb8KxN+o/KKW4Ui0KpoPeNoLasCC+CZAcQohCvcPWJeRd5UNsg==
+"@aptos-labs/aptos-names-connector@^0.0.2":
```

Comment:
Why these changes?

Reply:
last [PR](https://github.com/aptos-labs/explorer/pull/440) only updated `package.json` but not `yarn.lock`